lms/views/dashboard/api/user.py

```diff
@@ -23,6 +23,16 @@ class ListUsersSchema(PaginationParametersMixin):
     """Return users that belong to the assignment with this ID."""
 
 
+class UsersMetricsSchema(PaginationParametersMixin):
+    """Query parameters to fetch metrics for users."""
+
+    assignment_id = fields.Integer(required=True, validate=validate.Range(min=1))
```

Comment:
For now we only allow one assigment_id, same behaviour as the URL parameter.

Reply:
Ah I broke it when I removed the parent, class `PaginationParametersMixin` (this is not yet paginated) which had the right location attribute.

Sorry about that.

Reply:
No worries 🙂